lib/internal/Magento/Framework/DB/QueryInterface.php

```diff
@@ -44,7 +48,7 @@ public function getSize();
      * Get sql select string or object
      *
      * @param bool $stringMode
-     * @return string || Select
+     * @return string|Select
```

Comment:
I’d also revert such adjustments, because of which the SVC is failing, and requires a separate approval step.